packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -34,6 +34,7 @@ export const PRODUCT_JETPACK_SOCIAL_BASIC = 'jetpack_social_basic_yearly';
 export const PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY = 'jetpack_social_basic_monthly';
 export const PRODUCT_JETPACK_SOCIAL_ADVANCED = 'jetpack_social_advanced_yearly';
 export const PRODUCT_JETPACK_SOCIAL_ADVANCED_MONTHLY = 'jetpack_social_advanced_monthly';
+export const PRODUCT_JETPACK_STATS_YEARLY = 'jetpack_stats_yearly';
```

Comment:
It looks like further down in this file, on line 232, there's a `JETPACK_PRODUCTS_BY_TERM` array that Stats might be useful to go in now that there is a yearly and monthly term, and perhaps in the `JETPACK_PRODUCT_PRICE_MATRIX` array as well